core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java

```diff
@@ -33,10 +43,12 @@
  * Created by gupele on 3/30/2015.
  */
 public final class JniPCConnector {
+    public static final String AI_BASE_FOLDER = "AISDK";
+    public static final String AI_NATIVE_FOLDER = "native";
     public static final String PROCESS_SELF_INSTANCE_NAME = "__SELF__";
 
     private static final String BITS_MODEL_64 = "64";
-    private static final String NATIVE_LIBRARY_64 = "applicationinsights-core-native-win64";
+    private static final String NATIVE_LIBRARY_64 = "applicationinsights-core-native-win64.dll";
```

Comment:
For completeness please also add NATIVE_LIBRARY_32 (even we won't use it in this release)